pennylane/templates/broadcast.py

```diff
@@ -244,10 +305,79 @@ def circuit(pars):
                     return qml.expval(qml.PauliZ(0))
 
                 pars1 = [1, 2, 3]
+
                 circuit([pars1])
+
+        * Chain pattern with four wires (applying 3 blocks)
```

Comment:
Just fruit for thought with these examples: perhaps to grasp the power of `broadcast` and these `patterns`, an explicit equivalent circuit could also be included. In this case that would be
```python
                @qml.qnode(dev)
                def circuit(pars):
                    qml.CRot([0,1], parameters=pars1)
                    qml.CRot([1,2], parameters=pars2)
                    qml.CRot([2,3], parameters=pars3)
                    return qml.expval(qml.PauliZ(0))
```
This would also make understanding them easier and for example seeing which wire is the *control* and the *target* easier as well.

And for the `ring`:
```python
                @qml.qnode(dev)
                def circuit(pars):
                    qml.CRot([0,1], parameters=pars1)
                    qml.CRot([1,2], parameters=pars2)
                    qml.CRot([2,3], parameters=pars3)
                    qml.CRot([3,0], parameters=pars4)
                    return qml.expval(qml.PauliZ(0))
```

Reply:
I'll add that, good idea!